menus/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+
```

Comment:
please don't commit PYC files like the above

i think we have them added to our .gitignore so shouldnt be a problem in main repo